### ISSUE TITLE
Support for Templates

### ISSUE THREAD

**Issue by ChetanBhasin** [sebrichards/postmark-scala#3]
Can we have support for Postmark templates in this client?

**Comment by sebrichards** [comment]
Yeah sure... are you thinking just a new 'sendUsingTemplate()' method, or are you after full CRUD for templates?
